scripts/packaging/extensionsfolderpreupm.ps1

```diff
@@ -0,0 +1,80 @@
+<#
+.SYNOPSIS
+    Prepares the MRTK\Extensions folder for UPM packaging.
+.DESCRIPTION
+    Prepares the MRTK\Extensions folder for UPM packaging.
+.PARAMETER PackageRoot
+    The root folder containing the examples package contents. If not specified, the current folder is presumed.
+#>
+param(
+    [string]$PackageRoot
+)
+
+if (-not $PackageRoot) {
+    throw "Unknown package root path. Please specify -PackageRoot when building."
+}
+
+# This hashtable contains mappings of the sample categories to the folder which contains
+# the code and assets.
+#
+# Note that capitalization below in the key itself is significant.
+#
+# These paths are PackageRoot relative.

```

Comment:
This comment line doesn't seem to be accurate - it seems to be relative to something else.

Reply:
the paths are relative to PackageRoot. please see 

```
    $sampleFolder = $entry.Value
    Write-Output "Copying $PackageRoot\$sampleFolder to $samplesFolder\$sampleGroupName"
    Copy-Item -Path "$PackageRoot\$sampleFolder" -Destination "$samplesFolder\$sampleGroupName" -Recurse -Force
```

the capitalization significance is incorrectly stated